src/main/webapp/js/finscholar/collegelistview/collegelistview.js

```diff
@@ -15,21 +15,15 @@
 /** @fileoverview The mini controller for college list view. */
 
 goog.module('finscholar.collegelistview');
+const {BasicView} = goog.require('basicview');
 const {collegelist} = goog.require('finscholar.collegelistview.templates');
 
 /** The mini controller for college list view. */
-class CollegeListView {
-  
+class CollegeListView extends BasicView {
   constructor() {
-  }
-
-  /**
-   * Renders a college list view to the container.
-   * @param {!Element} container The HTML container to load the view.
-   */
-  renderView(container) {
-    container.innerHTML = collegelist();
+    super();
+    super.setCurrentContent(collegelist());
```

Comment:
Not going to force you to change this only because I know you're short on time, but this doesn't belong in the constructor. The render method should be dynamically calling the template. Ideally you would create another method in the base class like `getHtml` or similar that each subclass would override. That method would get and return the template html.